<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать книгу</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Основная информация</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_title">
        <property name="placeholderText">
         <string>Введите название книги</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Автор:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_author">
        <property name="placeholderText">
         <string>Введите автора</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Жанр:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_genre"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Статус:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_status">
        <item>
         <property name="text">
          <string>Хочу прочитать</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Читаю</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Прочитано</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Отложено</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата начала:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="date_start">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Дата окончания:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateEdit" name="date_finish">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Оценка:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radio_rating_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rating_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rating_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rating_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rating_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_rating_none">
          <property name="text">
           <string>Нет</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Количество страниц:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spin_pages">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Обложка:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_load_cover">
          <property name="text">
           <string>Загрузить изображение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_clear_cover">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Отзыв и заметки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="text_review">
        <property name="placeholderText">
         <string>Напишите ваш отзыв о книге...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_cover_preview">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>